pypit/arsort.py

```diff
@@ -978,7 +985,7 @@ def write_setup(setup_dict, use_json=False):
     # Write
     setup_file, nexist = get_setup_file()
     if nexist == 1:
-        msgs.error("Cannot over-write existing setup file.  If you wish to, remove and rerun")
```

Comment:
At this stage, I think it is fine to over-write.

If we want to be strict about `output overwrite True`, that should
be a separate PR and touch many aspects.